src/index.html

```diff
@@ -37,15 +37,15 @@ <h3>Recent Scores</h3>
             <!-- second column -->
             <div class="col-6 d-flex flex-column gap-3">
 
-                <h3>Add your score</h3>
+                <h3>Add Your Score</h3>
 
                 <form id="formData">
 
                     <input id="nameInput" class="form-control mb-3" type="text" placeholder="Your name">
 
                     <input id="scoreInput" class="form-control mb-3" type="text" placeholder="Your score">
 
-                    <button id="formButton" type="button" class="btn btn-outline-secondary">Submit</button>
+                    <button id="formButton" type="button" class="btn btn-outline-primary">Submit</button>
```

Comment:
- [ ] Kindly make this button be on the right hand side of the form as it is on the wireframe.
### Your work
![image](https://user-images.githubusercontent.com/86200356/216325166-177d4983-8488-4537-b8ba-c91a7d9a6be6.png)
### Wireframe
![image](https://user-images.githubusercontent.com/86200356/216325303-4a2c1062-8814-4d56-83fc-e8157930197d.png)